packages/extension/language-client/client.ts

```diff
@@ -101,9 +101,16 @@ export class SQLToolsLanguageClient implements SQLTools.LanguageClientInterface
         version: VSCodeVersion,
       },
     };
-    const selector = ConfigManager.completionLanguages
-      .concat(ConfigManager.formatLanguages)
-      .reduce((agg, language) => {
+    let selector = [];
+    if (ConfigManager.completionLanguages){
+      selector.concat(ConfigManager.completionLanguages);
```

Comment:
`array.concat` doesnt change the original array. So we need to assign it back to `selector` or use `push`.

`selector = selector.concat(ConfigManager.completionLanguages);`
or 
`selector.push(...ConfigManager.completionLanguages);`

Reply:
sure. stupid me...
was so happy it works, and no way to check the specific logic.